integration_tests/models/integration_tests_node.yml

```
@@ -0,0 +1,14 @@
+version: 2
+
+models:
+  - name: dummy_model
```

```suggestion
  - name: "var('integration_tests_dummy_model_name', 'dummy_model')"
```

`dbt run --vars "{'integration_tests_dummy_model_name':'dummy_model_2'}"`
`dbt build --vars "{'integration_tests_dummy_model_name':'dummy_model_2'}"`
bug in dbt-core preventing : `dbt test --vars "{'integration_tests_dummy_model_name':'dummy_model_2'}"`